weaver/core/drivers/fabric-driver/server/helpers/logger.ts

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
There's a logger for fabric-driver already present in `server/logger.ts` please use that instead of this.

Reply:
@sandeepnRES I see that we have already two files:  `server/helpers/logger.ts` and `server/logger.ts`. I checked their history and it seems they have been originally added by Rama and you. Not sure if I need to do anything here. I'll leave it for a future refactoring task for now.

Reply:
@outsidethecode no this `helpers` directory was added by you by copying from fabric-cli code, remember? because you needed that for asset lock functions.

Reply:
@outsidethecode Please remove `helpers/loggers.ts` and replace references to it in your code with `loggers.ts`. See my above comment: we should not have redundant code in the main branch. The process of a pull request (PR) is designed for this purpose: to give the submitted feedback about their code quality so they can improve and resubmit.